astrodb_utils/tests/test_publications.py

```diff
@@ -39,3 +47,75 @@ def test_ingest_publication_errors(db):
     assert " similar publication already exists" in str(error_message.value)
     # TODO - Mock environment  where ADS_TOKEN is not set. #117
 
+    ingest_publication(db, bibcode="2024ApJ...962..177B", ignore_ads=True)
+
+
+def test_ingest_publication(db):
+    ingest_publication(
+        db, reference="test05", bibcode="2024ApJ...962..177B", ignore_ads=True
+    )
+    assert find_publication(db, reference="test05")[0]  # True
+
+    ingest_publication(db, reference="test10", doi="10.1086/513700", ignore_ads=True)
+    assert find_publication(db, reference="test10")[0]  # True
+
+
+@pytest.mark.skipif(check_ads_token() is False, reason="ADS_TOKEN not set")
```

Comment:
This is OK (and clever) but in the future, we may want to consider using mock to simulate the return `ads.SearchQuery` to avoid live calls (and to control what is returned should that change)